backend/typescript/services/implementations/camperService.ts

```diff
@@ -12,7 +12,23 @@ class CamperService implements ICamperService {
   async createCamper(camper: CreateCamperDTO): Promise<CamperDTO> {
     let newCamper: Camper;
     let existingCamp: Camp | null;
+
     try {
+      existingCamp = await MgCamp.findById(camper.camp);
+
+      if (existingCamp) {
+        const existingCampObj = existingCamp.toObject();
+
+        if (
+          existingCamp &&
```

Comment:
do we need this second check for `existingCamp` if the check is already done in the if statement above?